st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -180,11 +180,11 @@ def _is_valid_node_name(self, all_node_names, node_name):
             # This task name needs to be resolved during run time so we cant validate the name now
             return True
 
-        for jinja_start_marker in JINJA_START_MARKERS:
-            if jinja_start_marker in node_name:
-                # This task name needs to be resolved during run time so we cant validate the name
-                # now
-                return True
+        is_jinja_expression = jinja_utils.is_jinja_expression(value=node_name)
```

Comment:
return jinja_utils.is_jinja_expression(value=node_name) should be sufficient, no?